app/src/play/java/com/amaze/filemanager/activities/MainActivity.java

```diff
@@ -631,7 +631,7 @@ private void initFabToSave(final ArrayList<Uri> uris) {
      * of possible deadlock situation and the asynchronous behaviour of LibSuperSU
      */
     private void initializeInteractiveShell() {
-        // only one looper can be associated to a thread. So we're making sure not to create new
+        // only one looper can be associated to a thread. So we are making sure not to create new
```

Comment:
This is a bit excessive...

Reply:
yeah ,don't like shorthand terms in docs/comments, gives a little pause  :)